docs/UserGuide.md

```diff
@@ -252,12 +271,21 @@ Parameters:
 <box type="warning" seamless>
 
 **Warnings**:
-- At least one non-empty parameter must be given
-- No duplicate parameter can be used
-- Find result(s) will contain/satisfy all the given parameters
-- Find feature is case-insensitive
+- At least one non-empty parameter must be given.
+- No duplicate parameter can be used.
+- A spacing between `find` and `-s` is compulsory.
```

Comment:
Maybe the phrasing can be changed to:
    At least one space is needed between `add` and `-s`.
to standardise with the other commands

Reply:
Ok finished standardising all warnings regarding spacing requirements